Resources/NetHook2/NetHook2/steam/net.h

```diff
@@ -0,0 +1,71 @@
+
+#ifndef STEAM_NET_H_
+#define STEAM_NET_H_
+#ifdef _WIN32
+#pragma once
+#endif
+
+
+#include "steamtypes.h"
+
+
+typedef uint32 HCONNECTION;
+
+
+enum EWebSocketOpCode
+{
+	k_eWebSocketOpCode_Continuation	= 0x00,

```

Comment:
Nit: 0x00 isn't vertically aligned with the others here.

Reply:
It's aligned with a tab character in VS, I blame github.